webpack/scenes/ContentViews/Details/ContentViewDetailActions.js

```diff
@@ -115,6 +117,25 @@ export const getContentViewFilters = (cvId, params) => get({
   url: api.getApiUrl('/content_view_filters'),
 });
 
+export const deleteContentViewFilters = (cvId, ids, handleSuccess) => put({
```

Comment:
Weren't you going to try `updateData` instead of passing `handleSuccess`? How did that go?

Reply:
I can explain later, long story short, it doesn't make sense to use updateData when deleting items from a list due to pagination.